### PR TITLE
Move Vagrant instructions lower in the README

### PR DESCRIPTION
**Pull request by ewpatton**
This PR adjusts the readme based on our discussion in today's dev meeting.

### CODE REVIEW

**Comment by ewpatton** [comment]
@AppInventorWorkerBee retest this please

File: README.md [hunk 1]
```diff
@@ -273,6 +219,49 @@ The release configuration sets the following additional options:
 
 Logs can be found at http://localhost:9876/log/ode and SourceMaps at http://localhost:9876/sourcemaps/ode
 
+## Setup Instructions (Vagrant)
+
+Using Vagrant for App Inventor development is a method of last resrt to get a development environment up and running is to use the provided Vagrantfile. Install [Vagrant](https://vagrantup.com) and open a terminal in the root directory of this repository. Run the following commands
```
Comment (jisqyv) [inline README.md:224]: s/resrt/resort/

File: README.md [hunk 2]
```diff
@@ -273,6 +219,49 @@ The release configuration sets the following additional options:
 
 Logs can be found at http://localhost:9876/log/ode and SourceMaps at http://localhost:9876/sourcemaps/ode
 
+## Setup Instructions (Vagrant)
+
+Using Vagrant for App Inventor development is a method of last resort to get a development environment up and running is to use the provided Vagrantfile. Install [Vagrant](https://vagrantup.com) and open a terminal in the root directory of this repository. Run the following commands
```
Comment (mark-friedman) [inline README.md:224]: This sentence doesn't quite parse.  Perhaps you could just delete "is to use the provided Vagrantfile"?

**Review by jisqyv — APPROVED** [review]
LGTM

**Pull request merged**